src/Mesh/babylon.subMesh.ts

```diff
@@ -247,6 +247,22 @@
         public intersects(ray: Ray, positions: Vector3[], indices: IndicesArray, fastCheck?: boolean): Nullable<IntersectionInfo> {
             var intersectInfo: Nullable<IntersectionInfo> = null;
 

```

Comment:
I don't think we should take that in account. The way meshes are drawn should not impact geometry intersections.

Reply:
It's not that simple. The drawMode changes the way indices are stored. There is also the case where there are no indices. My change will make it such that `intersects` will return null with mesh where fillMode is not triangles. Without this change will cause an exception. Making it work will require quite a bit of code.

Reply:
Makes sense!